packages/react/src/Accordion/AccordionContext.tsx

```diff
@@ -8,12 +8,12 @@ import { HeadingLevel } from '../Heading/Heading'
 
 export type AccordionContextValue = {
   headingLevel: HeadingLevel
-  section?: boolean
+  as?: 'div' | 'section'
```

Comment:
Can you reorder these and those for the default values?

Reply:
I renamed `as` to `sectionAs`, so it's in the right order again.